Gemfile

```diff
@@ -11,7 +11,7 @@ def location_for(place, fake_version = nil)
 end
 
 group :test do
-  gem 'voxpupuli-test', '>= 1.0.0',  :require => false
+  gem 'voxpupuli-test', '>= 1.4.0',  :require => false
```

Comment:
This is modulesynced.

Reply:
I know, I wanted a hotfix to get master green again. I will have a look at modulesync_config later :)